### PR TITLE
docs(ja): translate configuration-servermiddleware

### PR DESCRIPTION
**Pull request by aytdm**
resolve https://github.com/vuejs-jp/ja.nuxtjs.org/issues/136, resolve https://github.com/vuejs-jp/ja.nuxtjs.org/issues/127
ref: https://github.com/nuxt/nuxtjs.org/blob/master/content/ja/_archives/api/configuration-servermiddleware.md

### CODE REVIEW

File: content/ja/guides/configuration-glossary/configuration-servermiddleware.md [hunk 1]
```diff
@@ -1,73 +1,73 @@
...
-It is also possible to write custom middleware. For more information See [Connect Docs](https://github.com/senchalabs/connect#appusefn).
+カスタムミドルウェアの作成も可能です。詳細については [Connect Docs](https://github.com/senchalabs/connect#appusefn) を参照してください。
 
-Middleware (`api/logger.js`):
+ミドルウェア（`api/logger.js`）:
 
 ```js{}[api/logger.js]
 export default function (req, res, next) {
-  // req is the Node.js http request object
+  // req は Node.js の HTTPリクエストオブジェクトです
```
Comment (inouetakuya) [inline content/ja/guides/configuration-glossary/configuration-servermiddleware.md:64]:
```suggestion
  // req は Node.js の HTTP リクエストオブジェクトです
```

File: content/ja/guides/configuration-glossary/configuration-servermiddleware.md [hunk 2]
```diff
@@ -1,73 +1,73 @@
...
   console.log(req.url)
 
-  // res is the Node.js http response object
+  // res は Node.js の HTTPレスポンスオブジェクトです
```
Comment (inouetakuya) [inline content/ja/guides/configuration-glossary/configuration-servermiddleware.md:67]:
```suggestion
  // res は Node.js の HTTP レスポンスオブジェクトです
```

**Review by inouetakuya — CHANGES_REQUESTED** [review]
🙏

File: content/ja/guides/configuration-glossary/configuration-servermiddleware.md (hunk 1 above)
Comment (aytdm) [reply content/ja/guides/configuration-glossary/configuration-servermiddleware.md]: 🙈 
ありがとうございます、修正しました

**Review by inouetakuya — APPROVED** [review]
Thank you!

**Pull request merged**